ykrt/src/mt.rs

```diff
@@ -912,8 +913,8 @@ mod tests {
             match ctx.prog[ctx.pc] {
                 INC => {
                     ctx.pc += 1;
-                    if ctx.run == 0 {
-                        ctx.run = 99;
+                    if ctx.run > 0 {
```

Comment:
This feels a bit weird: why `>` and why `+= 99`?

Reply:
I needed to adjust the test to catch the bug. The `+=` is so that I can tell if `step_fn` ran twice. If the value is 100 it ran once, if its 199 it ran twice. That means I also had to change the condition so that the `if` triggers on both executions of `step_fn`.

Reply:
Silly question: are there smaller values which also highlight the bug, or is this the smallest (or close to the smallest) value?

Reply:
Any number will do. I don't know why I chose `99`. Want me to pick something smaller?

Reply:
Yes, that would probably be a good plan :) Also when we come to squash, I think it would be good to add a bit more information to the commit message about what the bug was and why this commit fixes it (but we can worry about this in the squashing phase).